### ISSUE TITLE
Add submission meta data to geojson export 

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-flow#1810]
In the implementation of exporting geoshapes we have did not include an identifier for the users to see which geoshape (and all other answers) belong to which data point. 

@joycarpediem created for a user the file with the identifiers separately to solve the problem but we need to think of a solution. Joy, could you please shed more light on the case (current situation)? 
Thank you

**Comment by joycarpediem** [comment]
UTZ has a registration form and a form for farm information.
Geoshape is collected as part of the farm information form.

In our current implementation of geojson export, we include all question text and responses as properties for a feature.
We do this because when you import a geojson into a GIS system, the properties are converted into columns. So now the user has all the data and the shape ( structurally it looks very similar to excel but here the shape is stored as a geometric object rather than a string) 

"type": "FeatureCollection",
	"properties": {
		"formId": 20730916,
		"questionId": 25721122,
		"questionText": "Définir les limites de la parcelle, utilisant le GPS"
	},
	"features": [{
		"type": "Feature",
		"properties": {
			"Point 1: Calcul sur la densité d'arbres: Quelle est la distance en mètres à l'arbre 2?": "2.5",
			"La protection contre l'érosion du sol est-elle suffisante?": "[{\"text\":\"Pas nécessaire\",\"code\":\"888\"}]",
			"Point 2: Y a-t-il une bonne pratique de la gestion de la récolte?": "[{\"text\":\"Preuve suffisante\",\"code\":\"3\"}]",....

What we do not do is add the meta-data like Identifier, submitter name, collection date etc to the properties.
The lack of Identifier means that the data form registration form and farm information cannot be combined.

Need:
Include metadata columns also as part of geojson properties


As a workaround , i download the rawdata spreadsheet for the farm information form and run a script which adds all the data (including metadata as properties)
Sample:

https://www.dropbox.com/s/ef4mnuo78lvtjea/geoshape_withQuestionID.json?dl=0

**Comment by janagombitova** [comment]
When making changes to the export we should also add the meta data of the polygon that we currently calculate:
- points 
- length 
- area

**Comment by janagombitova** [comment]
Agreed with Joy, Geert and Gunnet to the have the task of adding in the meta data as a support task (regular support work - not part of the extra budgeted hours) for now. In May we need to estimate how extensive this change might be to see if and how it can fit into the planning. I will connect with the partner requesting this change about Flow planning in May again.

**Comment by janagombitova** [comment]
@iperdomo and @dlebrero this is one of the smaller issues we spoke about yesterday

**Comment by dlebrero** [comment]
@joycarpediem @janagombitova  what is the metadata that needs to be included? Instance, Submission Date, Submitter, Duration?

**Comment by janagombitova** [comment]
@dlebrero This is the meta data we have in the excel exports: Identifier, Display Name, Device identifier, Instance, Submission Date, Submitter, Duration. I would suggest for consistency purposed to have them all also in the geoshape exports.

**Comment by dlebrero** [comment]
@janagombitova thanks! FYI the raw data exporter only has Instance, Submission Date, Submitter, Duration, maybe we want to make it also consistent

**Comment by kymni** [comment]
Testing on uat1 and geoshape report types returning null properties 

<img width="658" alt="screen shot 2018-07-05 at 14 07 25" src="https://user-images.githubusercontent.com/1155088/42320106-d3357092-805c-11e8-9a8c-88179152b0d5.png">

**Comment by dlebrero** [comment]
As per Skype chat, @kymni issue is not related to this change

**Comment by kymni** [comment]
confirmed working after #2767
